src/README.md

```diff
@@ -159,7 +159,7 @@ int main(int argc, char* argv[]) {
         // false means continue generation.
         return false;
     };
-    std::cout << pipe.generate("The Sun is yellow bacause", streamer);
+    std::cout << pipe.generate("The Sun is yellow bacause", ov::genai::streamer(streamer));
```

Comment:
This isn't the only problem in README.md. For example, `num_groups` doesn't exist anymore. Will it be fixed in the next PR?

Reply:
thanks for noticing this. i will fix it now

Reply:
I missed that in docs. Thanks again for pointing this out. 
renames `num_groups -> num_beam_groups; group_size -> num_beams / num_beam_groups` in all docs. Now changes are final